packages/discord.js/typings/index.d.ts

```diff
@@ -1070,6 +1070,7 @@ export class Emoji extends Base {
   public name: string | null;
   public get identifier(): string;
   public get url(): string | null;
+  public getURL(ext: ImageExtension): string;
```

Comment:
```suggestion
  public getURL(ext: ImageExtension): string | null;
```
as per previous suggestion the method can return null